frontend/src/components/MapView/DateSelector/index.tsx

```diff
@@ -335,10 +325,11 @@ const DateSelector = memo(
           justify="center"
           className={classes.datePickerContainer}
         >
+          {/* Mobile */}
           <Grid item xs={12} sm={1} className={classes.datePickerGrid}>
             <Hidden smUp>
               <Button onClick={decrementDate}>
-                <ChevronLeft />
+                <ChevronLeft style={{ color: '#101010' }} />
```

Comment:
It might be worth extracting some of these colors into the theme?

Reply:
How do I do that without possibly breaking other component current color ? 
I could add a `color.secondary` item with that color but it would not reflec actual application theme